core/new-gui/src/app/workspace/service/dynamic-schema/schema-propagation/schema-propagation.service.ts

```diff
@@ -218,6 +218,16 @@ export class SchemaPropagationService {
       if (v.additionalEnumValue) {
         attrNames.push(v.additionalEnumValue);
       }
+      if (v.title && !operatorSchema.jsonSchema.required?.includes(v.title)) {
+        if (v.default) {
```

Comment:
I think it's better to add comments to explain what is this block of code doing and why we need this workaround. Please do mention the issue https://github.com/ajv-validator/ajv/issues/1471